src/index.html

```diff
@@ -4,9 +4,305 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <div>
+        <a href="#home">
+          <img src="images/logo.png" alt="logo" class="logo">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="http://" class="is-active">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="http://">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="http://">
+              Смартфоны и телефоны
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="http://" data-qa="nav-hover">
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="http://">
+              Гаджеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="http://">
+              Планшеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="http://">
+              Фото
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="http://">
+              Видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="cardContainer">
+      <div class="card" data-qa="card">
+        <img class="card__image"
+          src="./images/imac.jpeg"
+          alt="imac image"
+        >
+        <div class="card__prodName">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</div>
+        <div class="card__prodSku">Код товара: 195434</div>
+        <div class="starsPlusReviews">
+          <div class="starsPlusReviews__stars">
+            <div class="stars starsPlusReviews__stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+          </div>
+          <div class="starsPlusReviews__prodReviews">Отзывов: 5</div>
+        </div>
```

Comment:
Check the 1st point from the [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)

Reply:
@Olexandr6 i checked it and..........